### ISSUE TITLE
Not all calendars are shown sometimes

### ISSUE THREAD

**Issue by massimilian0** [wernerjoss/grav-plugin-fullcalendar#58]
Hi,

A weird issue happens when visiting the calendar page.
Some (random) calendars are not shown on random ways.
This randomness changes on each refresh.
On firefox this happens more then on chrome based browsers.
I have 7 ics files stored in the user/data/calendars folders.

Thank you.

Kind regards,
Massimiliano

**Comment by wernerjoss** [comment]
you are right, this happens from time to time, a page reload fixes this here, though.
I have not yet found out a reason for this behaviour, maybe I'll look into this once again, but this will not happen in the near future, as I have switched to [Hugo](https://gohugo.io) in the meantime.
